_data/standing_meetings.yml

```diff
@@ -21,6 +21,10 @@
       date: 2022-06-15 14:00:00 +0200
       speaker: "Ioana Ifrim"
       link: "[Slides](/assets/presentations/CaaS_Weekly_15_06_2022_Ioana-Python_and_C++_clang-repl_integration.pdf)"
+    - title: "Clang-REPL Interactive Analysis - Roadmap"
```

Comment:
Your presentation should go after the one by Ioana.

Reply:
Done!